cmd/access/node_builder/access_node_builder.go

```diff
@@ -676,9 +677,13 @@ func (builder *FlowAccessNodeBuilder) Initialize() error {
 	// enqueue the regular network
 	builder.EnqueueNetworkInit()
 
-	builder.AdminCommand("get-transactions", func(conf *cmd.NodeConfig) commands.AdminCommand {
-		return storageCommands.NewGetTransactionsCommand(conf.State, conf.Storage.Payloads, conf.Storage.Collections)
-	})
+	builder.
+		AdminCommand("get-transactions", func(conf *cmd.NodeConfig) commands.AdminCommand {
+			return storageCommands.NewGetTransactionsCommand(conf.State, conf.Storage.Payloads, conf.Storage.Collections)
+		}).
+		AdminCommand("read-execution-data", func(conf *cmd.NodeConfig) commands.AdminCommand {
+			return stateSyncCommands.NewReadExecutionDataCommand(builder.ExecutionDataService)
```

Comment:
what is this for?

Reply:
Right now that's only available on ENs. Adding it to ANs so we can get access to execution data there too. It came up as useful when attempting to backfill some missing data from gcp